Tests/PSModuleGallery.Type.Tests.ps1

```diff
@@ -474,7 +558,46 @@ InModuleScope 'PSDepend' {
             function Install-Package {[cmdletbinding()]param( $Source, $Name, $RequiredVersion, $Force)}
             function Get-PackageSource { @([pscustomobject]@{Name = 'chocolatey'; ProviderName = 'chocolatey'}) }
             
-            It 'Runs Get-Package and Find-Package, skips Install-Package' {
+            It 'Skips Install-Package' {
+
+                Mock Install-Package
+                Mock Get-Package {
+                    [pscustomobject]@{
+                        Version = '1.1'
+                    }
+                }
+                Mock Find-Package
+
+                Invoke-PSDepend @Verbose -Path "$TestDepends\package.sameversion.depend.psd1" -Force -ErrorAction Stop
+
+                Assert-MockCalled Get-Package -Times 1 -Exactly
+                Assert-MockCalled Find-Package -Times 0 -Exactly
+                Assert-MockCalled Install-Package -Times 0 -Exactly
+            }
+        }
+
+        Context 'Latest package required, and already installed' {    
+
+            <#
+                This test works on my machine but not in AppVeyor (!)
```

Comment:
This was a really frustrating issue. If anyone can shed light on what the problem is that would be great.

Given that the test does pass, I strongly suspect this is a problem with the test and NOT a problem with the code being tested.